vdom/src/com/vdom/core/Game.java

```diff
@@ -2525,6 +2522,9 @@ protected void initCards() {
                     card = Cards.virtualKnight;
                 }
 
+                //TODO SPLITPILES add names for SplitPiles
```

Comment:
This is unnecessary since the split pile names will be parsed correctly by the existing code.